web/modules/custom/configform/configform.info.yml

```diff
@@ -0,0 +1,6 @@
+name: Form Tutorial
+type: module
+package: Custom Module
+description: Form Api Tutorial
+core_version_requirement: ^10
+configure: Form.configForm
```

Comment:
`config_form.settings`
Generally modules provide only 1 setting and the setting name should be `module_name.settings`.